atest/testdata/standard_libraries/process/terminate_process.robot

```diff
@@ -96,10 +96,11 @@ Terminate all processes
     END
 
 Terminating all empties cache
+    [Documentation]    FAIL Non-existing index or alias '1'.
     Some process
+    Switch process  ${1}
     Terminate All Processes    kill=True
-    ${handle} =    Some Process
-    Should Be Equal    ${handle}    ${1}
+    Switch process  ${1}
```

Comment:
Good idea to validate that `Switch process    1` (`1` should work in addition to `${1}`) works before cache is cleared, but now it's possible that the test fails already there and test status validation doesn't detect that. To make sure it fails only at the end, we'd either need to validate that on the `atest/robot` side or change the test a bit. The latter is probably easier. Could, for example, start two processes and then use `Switch process    2` before clearing the cache.

Reply:
updated the test case as it creates 2 processes then clears cache